actions/run_checkqc.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: get_response
```

Comment:
I think that the name of this action should be `run_checkqc`, and then the name of the python script should probably be the same to make it easier to see which things belong together.